### PR TITLE
🦉 Configure JReleaser via its Maven plugin

### PR DESCRIPTION
**Pull request by aalmiray**
Fix for #26

### CODE REVIEW

**Comment by aalmiray** [comment]
Converting to draft as JReleaser `0.6.0` is needed before this PR can be merged.

File: pom.xml [hunk 1]
```diff
@@ -306,5 +319,49 @@
...
+                <distributions>
+                  <kcctl>
+                    <name>kcctl</name>
+                    <type>NATIVE_IMAGE</type>
+                    <artifacts>
+                      <artifact>
+                        <path>{{artifactsDir}}/{{distributionName}}-{{projectVersion}}-linux-x86_64.zip</path>
+                        <transform>artifacts/{{distributionName}}-{{projectEffectiveVersion}}-linux-x86_64.zip</transform>
+                        <platform>linux-x86_64</platform>
+                      </artifact>
```
Comment (gunnarmorling) [inline pom.xml:346]: Isn't this missing the TAR.GZ flavour?

Comment (aalmiray) [reply pom.xml:346]: Yes. Are we producing one?

Comment (aalmiray) [reply pom.xml:336]: According to `src/main/assembly/assembly.xml` we only produce ZIP files for distributions.

Comment (gunnarmorling) [reply pom.xml:336]: Hum, that's really weird. You are right, but I could swear I configured it recently to produce both TAR.GZ and ZIP.

Comment (aalmiray) [reply pom.xml:336]: Wans't that the case for kcetcd?

Comment (gunnarmorling) [reply pom.xml:336]: Ah yes, of course. Ok, so at least I'm not dreaming :) Could you update the PR so we create and publish both formats?

Comment (aalmiray) [reply pom.xml:336]: OK, so you want kcctl to publish 6 artifacts in total?

**Comment by gunnarmorling** [comment]
> OK, so you want kcctl to publish 6 artifacts in total?

Yeah, or do you believe that's overkill? I always like leaving people the
choice between the archive format they prefer. E.g. imposing ZIP on Linux
users seems a bit odd. WDYT?

>

**Comment by aalmiray** [comment]
No, not really. However the inverse is true, l: offer tar.gz for the other platforms. Even though the build can produce 6 distro files only 3 may be released: 2 zips, 1 tar. WDYT?

**Comment by gunnarmorling** [comment]
> 2 zips, 1 tar. WDYT?

So ZIP for Windows and Mac, and TAR.GZ for Linux? That'd work for me, too.

**Comment by aalmiray** [comment]
OK. Will send another PR first that targets the current state. Then rebase this PR after merge.

**Comment by aalmiray** [comment]
Depends on https://github.com/kcctl/kcctl/pull/29

File: pom.xml [hunk 2]
```diff
@@ -36,11 +36,24 @@
...
+  <description>kcctl -- A CLI for Apache Kafka Connect</description>
+  <inceptionYear>2021</inceptionYear>
+  <url>https://github.com/kcctl/kcctl</url>
+
+  <developers>
+    <developer>
+      <id>gunnarmorling</id>
+      <name>Gunnar Morling</name>
+    </developer>
+  </developers>
```
Comment (gunnarmorling) [inline pom.xml:52]: Wanna add yourself too? Seems more than justified :)

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Applied, thx! If you want to add yourself as a developer as suggested in my last comment, just push a separate commit for that to main.

**Pull request merged**